src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/CreateAccessRequirementButton.java

```diff
@@ -126,11 +127,17 @@ private void useSrcWizard() {
     CreateOrUpdateAccessRequirementWizard wizard =
       ginInjector.getCreateOrUpdateAccessRequirementWizard();
 
-    CreateOrUpdateAccessRequirementWizardProps.OnComplete onComplete = () -> {
-      wizard.setOpen(false);
-      refreshCallback.invoke();
-      view.clearWidgets();
-    };
+    CreateOrUpdateAccessRequirementWizardProps.OnComplete onComplete =
+      accessRequirementID -> {
+        wizard.setOpen(false);
+        view.clearWidgets();
+        AccessRequirementPlace target = new AccessRequirementPlace("");
```

Comment:
Instead of refreshing the page, go to the AR place (pointing directly to this specific AR ID).